features/step_definitions/individual_steps.rb

```diff
@@ -1248,6 +1248,44 @@ class VlpDocument
   visit "/families/home"
 end
 
+Then(/^.+ should (.*) an American Indian or Alaska Native status$/) do |is_outstanding|
+  if is_outstanding == 'not see'
+    expect(page).not_to have_content("American Indian Status")
```

Comment:
Just noting-- Ideally we wouldn't be checking for hard coded content like this, but I realize we will be overhauling the UI shortly anyways and adding a unique `id` to the existing partial is not worth the trouble.

Reply:
I agree, for the moment I at least modified them to use the constants of American Indian Status: https://github.com/ideacrew/enroll/pull/4774/commits/6d80ef089df5eedc559b94e5d2342b53c0eb98c5#diff-b911f5fd6f4a943e31c8c93c7ed13793a5f223a7dd19a1d761712998594a6fdcR1253-R1255